contracts/FyToken.sol

```diff
@@ -1,8 +1,8 @@
 /* SPDX-License-Identifier: LGPL-3.0-or-later */
-pragma solidity ^0.7.0;
+pragma solidity ^0.8.0;
 
 import "@paulrberg/contracts/access/Admin.sol";
-import "@paulrberg/contracts/math/CarefulMath.sol";
+import "@paulrberg/contracts/math/Exponential.sol";
```

Comment:
Where's Exponential being used in this fuke?

Reply:
Good catch.